src/G2/Config/Config.hs

```diff
@@ -56,6 +61,7 @@ data Config = Config {
     , extraDefaultMods :: [FilePath]
     , logStates :: LogMode -- ^ Determines whether to Log states, and if logging states, how to do so.
     , sharing :: Sharing
+    , instTV :: InstTV -- allow the instantiation of types in the beginning or it's instantiate symbolically by functions  
```

Comment:
Might be a word missing here or something ("it's instantiate symbolically by functions"?)